Cargo.toml

```diff
@@ -55,10 +55,10 @@ doc-comment = "0.3"
 wasm-bindgen-test = "0.2"
 
 [package.metadata.docs.rs]
-all-features = true
+features = ["serde"]
```

Comment:
To the best of my knowledge, docs.rs always builds the docs with the nightly compiler. This probably doesn't need to be changed for *functional* reasons.

Reply:
fair! I mostly just didn't want to accidentally document too much, as the later commits in this PR help prevent.